magda-web-client/src/Components/Dataset/Add/AddDatasetLinkSection.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import {
+    File,
```

Comment:
Possibly if these are no longer "Files" we should name them to something that takes into account that it could be either a file or a service, like "Distribution"

Reply:
👍 The name looks confusing for me as well 😄 
Will change it to `Distribution`~